test/test_response.py

```diff
@@ -165,6 +165,57 @@ def test_chunked_decoding_gzip(self):
         assert r.read() == b''
         assert r.read() == b''
 
+    def test_decode_gzip_multi_member(self):
+        import zlib
+        compress = zlib.compressobj(6, zlib.DEFLATED, 16 + zlib.MAX_WBITS)
+        data = compress.compress(b'foo')
+        data += compress.flush()
+        data = data * 3
+
+        fp = BytesIO(data)
+        r = HTTPResponse(fp, headers={'content-encoding': 'gzip'})
+
+        assert r.data == b'foofoofoo'
+
+    def test_decode_gzip_error(self):
+        import zlib
```

Comment:
Do we need this line or the one that follows in this test?

Reply:
Ah I forgot to check whether this line is still useful after implementing it.  Will remove it.